CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for showing errors in child objects
```

Comment:
I would argue that this is a bugfix so you can use the `Fixed` title. 
And I think nested fields would be more accurate than child objects.

Reply:
hehe i dont wanted to blame the current implementation, thats why i added it as new feature but i would agree, is changed